### PR TITLE
Add ASaiM-MT and update Galaxy to 20.09

### PR DESCRIPTION
**Pull request by shiltemann**

### CODE REVIEW

**Comment by shiltemann** [comment]
@bgruening The Docker is [failing](https://quay.io/repository/bebatut/asaim-framework/build/baef8169-ac90-4b22-849c-6b8be2acd7f8) with a user authentication error when installing workflows, any ideas? Anything change in the Galaxy docker image recently ("recently" since this last succeeded in 2018 ;P) that might cause this or?

```
A build step failed: The command '/bin/sh -c startup_lite && galaxy-wait && workflow-install --workflow_path 
$GALAXY_ROOT/workflows/ -g http://localhost:8080 -u $GALAXY_DEFAULT_ADMIN_USER -p 
$GALAXY_DEFAULT_ADMIN_PASSWORD' returned a non-zero code: 1

Traceback (most recent call last): 
File "/tool_deps/_conda/bin/workflow-install", line 10, in <module> sys.exit(main()) 
File "/tool_deps/_conda/lib/python3.7/site-packages/ephemeris/workflow_install.py", line 46, in main import_workflow(gi, os.path.join(args.workflow_path, file_path), args.publish_workflows) 
File "/tool_deps/_conda/lib/python3.7/site-packages/ephemeris/workflow_install.py", line 18, in import_workflow existing_uuids = [d.get('latest_workflow_uuid') for d in gi.workflows.get_workflows()] 
File "/tool_deps/_conda/lib/python3.7/site-packages/bioblend/galaxy/workflows/__init__.py", line 47, in get_workflows workflows = self._get(params=params) 
File "/tool_deps/_conda/lib/python3.7/site-packages/bioblend/galaxy/client.py", line 129, in _get r = self.gi.make_get_request(url, params=params) 
File "/tool_deps/_conda/lib/python3.7/site-packages/bioblend/galaxyclient.py", line 67, in make_get_request params['key'] = self.key 
File "/tool_deps/_conda/lib/python3.7/site-packages/bioblend/galaxyclient.py", line 229, in key raise Exception("Failed to authenticate user.") 
Exception: Failed to authenticate user. 
```

File: Dockerfile [hunk 1]
```diff
@@ -49,11 +57,15 @@ ADD https://raw.githubusercontent.com/galaxyproject/training-material/master/top
...
+
+COPY create_admin_user.sh $GALAXY_ROOT/create_admin_user.sh
+
 ENV GALAXY_CONFIG_TOOL_PATH=/galaxy-central/tools/
+ENV PATH="${PATH}:/tool_deps/_conda/bin"
 RUN startup_lite && \
     galaxy-wait && \
-    workflow-install --workflow_path $GALAXY_ROOT/workflows/ -g http://localhost:8080 -u $GALAXY_DEFAULT_ADMIN_USER -p $GALAXY_DEFAULT_ADMIN_PASSWORD
+    $GALAXY_ROOT/create_admin_user.sh  && \
+    workflow-install --workflow_path $GALAXY_ROOT/workflows/ -g http://localhost:8080 -u $GALAXY_DEFAULT_ADMIN_EMAIL -p $GALAXY_DEFAULT_ADMIN_PASSWORD
```
Comment (bgruening) [inline Dockerfile]: @shiltemann does it work if you use the API key `fakekey` and maybe even the `--publish` flag?

Comment (bgruening) [reply Dockerfile]: https://ephemeris.readthedocs.io/en/latest/commands/workflow-install.html

Comment (shiltemann) [reply Dockerfile]: looks like it worked after I added the step to create the admin user https://quay.io/repository/bebatut/asaim-framework

publishing the workflows is probably nice to have yeah :+1:

**Pull request merged**